query/src/interpreters/interpreter_role_grant.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright 2021 Datafuse Labs.
```

Comment:
2022

Reply:
We can consider removing this time uniformly from license, should be in another PR.